packages/frontend/web/pages/Article.tsx

```diff
@@ -172,43 +188,45 @@ export const Article: React.FC<{
                 edition={data.CAPI.editionId}
             />
         </div>
-
-        <main className={bodyAdStyles}>
-            <Container borders={true} className={articleContainer}>
-                <article>
-                    <ArticleBody CAPI={data.CAPI} config={data.config} />
-                    <div className={secondaryColumn}>
-                        <div className={adSlotWrapper}>
-                            <AdSlot
-                                asps={namedAdSlotParameters('right')}
-                                config={data.config}
-                                className={stickyAdSlot}
-                            />
-                        </div>
+        <Container
+            borders={true}
+            className={cx(articleContainer, articleBodyAdStyles)}
```

Comment:
The class `articleBodyAdStyles` here seems at odds with the element it's added to - `ArticleBody` being further down the tree?

Reply:
I should probably rename the style... Coming PR